teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    return unless @type == age or @type == @parent_permission
+  end
```

Comment:
- [ ] In the `Teacher` class, kindly override the can_use_services? method to always return true. 

```ruby
def can_use_services?
true
end
````